biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -260,15 +262,16 @@ BAM alerts require BAM tools to be enabled.
 ### Configure BizTalk TMS
 
 >[NOTE]
->"Enterprise Single Sign-On Service"(SSO) is major dependency for TMS, TMS should be configured on server which has SSO service running. Also TMS is recommanded to configure only on one server in a BizTalk Group.
+>TMS should be configured on server which has SSO service running. Also TMS is recommanded to be configured only on one server in a BizTalk Group.
```

Comment:
IMHO these should be 2 separate bullet points. Please see my earlier comment

Reply:
Also, at least the first point should have IMPORTANT as TMS cannot work without it